packages/default/scss/input/_layout.scss

```diff
@@ -341,19 +341,25 @@
         @include border-radius( 0 !important ); // sass-lint:disable-line no-important
         border-width: $kendo-input-border-width 0;
         border-top-color: transparent !important; // sass-lint:disable-line no-important
-        background: none !important; // sass-lint:disable-line no-important
+        background-image: none !important; // sass-lint:disable-line no-important
 
-        .k-input-inner {
-            padding-left: 0 !important; // sass-lint:disable-line no-important
-            padding-right: 0 !important; // sass-lint:disable-line no-important
+        &:focus,
+        &.k-focus {
+            @include border-radius( $kendo-input-flat-border-radius !important ); // sass-lint:disable-line no-important
+        }
+        &:focus-within {
+            @include border-radius( $kendo-input-flat-border-radius !important ); // sass-lint:disable-line no-important
         }
```

Comment:
why don't we combine these rules instead of having them separately:
```scss
&:focus,
&:focus-within,
&.k-focus {
    @include border-radius( $kendo-input-flat-border-radius !important ); // sass-lint:disable-line no-important
}

```